benchmarks/python/test_huggingface_attn_bwd.py

```diff
@@ -95,3 +101,34 @@ def test_huggingface_attn_bwd_benchmark(
 
     if not disable_benchmarking:
         run_benchmark(benchmark, fd.execute, [grads, attn, dropout_mask])
+
+
+@pytest.mark.parametrize("compile", [False, True], ids=["eager", "compile"])
+@pytest.mark.parametrize("size", generate_attn_inputs())
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES)
+def test_huggingface_attn_bwd_baseline_benchmark(
+    benchmark,
+    size: tuple,
+    dtype: torch.dtype,
+    compile: bool,
+):
+    clear_cuda_cache()
+
+    batch_size, seq_len, nh, n_embd = size
+    dropout_p = 0.0
+    inputs = torch.randn(batch_size, nh, seq_len, seq_len, device="cuda", dtype=dtype)
+    attention_mask = torch.zeros(
+        batch_size, nh, seq_len, seq_len, device="cuda", dtype=dtype
+    )
+    attn = (inputs + attention_mask).view(batch_size * nh, seq_len, seq_len)
+    attn = torch.nn.functional.softmax(attn, dim=-1)
+    output = torch.nn.functional.dropout(attn, p=dropout_p)
+
+    grads = torch.randn(batch_size * nh, seq_len, seq_len, device="cuda", dtype=dtype)
+
+    run_benchmark(
+        benchmark,
+        torch.compile(unary_bwd_torch) if compile else unary_bwd_torch,
+        [output, grads],
+        iobytes=huggingface_attn_bwd_iobytes(size, dtype),
```

Comment:
can you comment why we need to set `iobytes`? what's the difference is using default.

Reply:
This is for uniformity of IObytes between nvFuser and torch when the inputs/outputs differ between the two. See the discussion at: https://github.com/NVIDIA/Fuser/pull/1725. I'll add a reference to this PR in a comment.